js/package.json

```diff
@@ -1,28 +1,28 @@
 {
-  "name": "@zondax/ledger-dfinity",
+  "name": "@zondax/ledger-icp",
   "author": "Zondax GmbH",
   "license": "Apache-2.0",
   "version": "0.1.0",
-  "description": "Node API for Dfinity App (Ledger Nano S/X)",
+  "description": "Node API for InternetComputer App (Ledger Nano S/X)",
```

Comment:
```suggestion
  "description": "Node API for the Internet Computer App (Ledger Nano S/X)",
```

Reply:
@ielashi any advice/opinions on the name?

Reply:
@jleni I'd go with "the Internet Computer App" per your suggestion. Everything else looks great!